spec/models/manageiq/providers/infra_manager_spec.rb

```diff
@@ -1,16 +1,28 @@
-describe EmsInfra do
-  it ".types" do
-    expected_types = [ManageIQ::Providers::Vmware::InfraManager, ManageIQ::Providers::Redhat::InfraManager, ManageIQ::Providers::Microsoft::InfraManager, ManageIQ::Providers::Openstack::InfraManager].collect(&:ems_type)
-    expect(described_class.types).to match_array(expected_types)
-  end
+describe ManageIQ::Providers::InfraManager do
+  describe ".ems_timeouts" do
+    before do
+      stub_settings(:ems => {:ems_amazon => {},
+                             :ems_redhat => {:inventory => {:read_timeout => 5.hours}}})
```

Comment:
This should be stubbed as the string `"5.hours"` which is how people will likely enter it in the config.